index.js

```diff
@@ -0,0 +1,22 @@
+const express = require("express");
+const cors = require("cors");
+
+const auth = require("./auth/middleware");
+const corsMiddleware = cors();
+
+const app = express();
+const port = process.env.PORT || 5000;
+
+const authRouter = require("./auth/router");
+const userRouter = require("./user/router");
+const menuRouter = require("./menu/router");
```

Comment:
Nice that you are trying to order this, there is an order for both front and back end:
1. Libraries and modules
2. my other files (routers, models) or (components, actions)
3. Css files